api/src/pcapi/core/search/__init__.py

```diff
@@ -655,39 +657,79 @@ def unindex_all_venues() -> None:
         logger.exception("Could not unindex all venues")
 
 
-def reindex_offers_if_ean_booked_since(offer_ids: list, since: datetime.datetime) -> None:
-    """
-    This function loads all offers booked the last x hours and
-    if they have an ean, it will reindex all related offers with the same ean
-    """
+class Memoize:
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Ce n'est pas utilisé